shared/teams/main/team-list.js

```diff
@@ -65,9 +73,13 @@ const Row = ({name, membercount, isNew, newRequests, onOpenFolder, onManageChat,
           </Box>
         </Box>
       </ClickableBox>
-      {!isMobile && <Icon type="iconfont-folder-private" onClick={onOpenFolder} />}
+      {!isMobile && <Icon type="iconfont-folder-private" onClick={onOpenFolder || undefined} />}
```

Comment:
why is this || undefined required?

Reply:
It was to avoid a flow error passing a null onClick.  Decided to fix this a different way, though:

The reason that onOpenFolder and onManageChat can be omitted is that when you create a subteam, it'll appear in the team's subteam rows but you *can't* access the KBFS folder or manage channels for it.  Because you aren't a member of that subteam you just created yet.  (KBFS'll throw permission denied if you try to cd, Chat RPC will throw an error.)

Instead of disabling the icon in that case by passing in no onClick, I'm just going to avoid showing the two icons completely.  So when you create a subteam you'll have a row without those two icons, and when you click on it you're prompted to join the team, and then they'll show up.  Make sense?

Reply:
I'd just update icon.js.flow to let onClick?: ?() =>  so you can pass null to it